dFusion/dFusionSpec.md

```diff
@@ -79,11 +79,92 @@ To summarize, here is a list of state that is stored inside the smart contract:
 ## Batch Auction Workflow
 
 The trading workflow consists of the following sequential processes:
+
+0. Account opening, deposits & withdrawals
 1. On-Chain order collection
 2. Finding the batch price: optimization of batch trading surplus (off-chain)
 3. Verifying batch price and trade execution (zkSnark)
 4. Restart with step 1
 
+### Account opening & token registration
+
+
+#### Registering accounts
+The contract specifies a constant number of accounts, `K`, that can be opened by anyone by providing an `accountIndex` such that `1 <= accountIndex <= K`. 
+The account index is referred to, synonomously, as the account ID.
```

Comment:
synonymously? pseudonymously?

Reply:
At some point it will likely be only one. I prefer `accountId` because Indexes generally start at zero while IDs start at one (as in our situation).

Reply:
I will use interchangeably for now.